FLIR/conservator/wrappers/video.py

```diff
@@ -3,14 +3,26 @@
 from FLIR.conservator.generated.schema import Query, Mutation
 from FLIR.conservator.paginated_query import PaginatedQuery
 from FLIR.conservator.wrappers.frame import Frame
-from FLIR.conservator.wrappers.media import MediaType
+from FLIR.conservator.wrappers.media import MediaType, MediaCompare
```

Comment:
Same comment as in image.py; the verify_md5() method is what returns a MediaCompare object, but the source for this module doesn't instantiate the class itself.